source/js/main.js

```diff
@@ -379,6 +379,96 @@ let main = {
       );
     });
 
+    function profileDirectoryFilterAnalytics() {
```

Comment:
we don't ever call this code more than once, so in this case let's not wrap this in a function.